skrub/_gap_encoder.py

```diff
@@ -198,6 +199,77 @@ def _init_w(self, V: NDArray, X) -> tuple[NDArray, NDArray, NDArray]:
         B = A.copy()
         return W, A, B
 
+    def _minibatch_convergence(self, batch_size, batch_cost, n_samples, step, n_steps):
```

Comment:
Useful for readability and IDE support, even on private API.
```suggestion
    def _minibatch_convergence(self, batch_size: int, batch_cost: float, n_samples: int, step: int, n_steps: int):
```